api/tests.py

```diff
@@ -1,3 +1,6 @@
 from django.test import TestCase
 
-# Create your tests here.
+
+class ApiTest(TestCase):
```

Comment:
As a suggestion, you can test if circleCI works without the necessity of push a commit, check https://circleci.com/docs/2.0/local-cli/ 😄